shell/src/main/java/alluxio/cli/docgen/ConfigurationDocGenerator.java

```diff
@@ -119,9 +155,19 @@ public static void writeCSVFile(Collection<? extends PropertyKey> defaultKeys, S
     } finally {
       try {
         closer.close();
+        if (validate) {
+          compareFiles(CSV_FILE_NAMES, filePath, "CSV");
+        }
       } catch (IOException e) {
         LOG.error("Error while flushing/closing Property Key CSV FileWriter", e);
```

Comment:
this log message is not accurate anymore after adding additional code in the try block, should be updated. same for the yml files section